br-new

```diff
@@ -0,0 +1,230 @@
+br:
+  profile_help: |
+    A gloriosa substituição do fc! Confira !!perfil de ajuda para começar.
+  
+    Tente também !!perfil fileira de ajuda para informações mais detalhadas.
+  
+    Você tem que adicionar seu fc antes que seu perfil funcione!
+  
+    Por favor, marque "!!perfil ajuda fc" e adicione seu código de amigo antes de dizer que spyke não está funcionando! Obrigado!
+
+  level_help: |
+    
+    Defina seu nível em splatoon
+    Exemplo:
+    !!perfil nivel 50
+    Se você é um jogador de estrela, adicione um S no final após o seu nível
+    !!perfil nivel 50 S
+  level_set: seu nível foi ajustado para
+
+  level_error1: você não pode ser nível negativo ..
+
+  level_error2: Vamos lá, isso não é nem um número!
+
+  level_error3: mate, o nível 99 é o máximo ..
+
+  level_error4: U WOT MATE ?! S é o único nível que eu aceito!
+
+    weapon_help: |
```

Comment:
In addition to the key, check your indentation too. This is why a program such as Notepad++ is useful if not essential
```suggestion
  pf_wep: |
```